backend/src/Parser/Parser.fs

```diff
@@ -42,7 +43,10 @@ module DType =
     // with type args
     | "List", [ arg ] -> PT.TList(fromSynType arg)
     | "Option", [ arg ] -> PT.TOption(fromSynType arg)
-
+    | "Result", [ okArg; errorArg ] ->
+      PT.TResult(fromSynType okArg, fromSynType errorArg)
+    | "Tuple", first :: second :: theRest ->
```

Comment:
This surprises me - is the F# compiler using the word "Tuple"?

Reply:
Hmm
The context here is handling `Json.serialize<Tuple<int, string>>`.
I guess I could have done `Json.serialize<int * string>`, as F# might "expect".